include/fmt/compile.h

```diff
@@ -529,6 +532,17 @@ template <typename Char> struct code_unit {
   }
 };
 
+// This ensures that the argument type is convertile to `const T&`.
+template <typename T, int N, typename... Args>
+constexpr const T& get_arg_checked(const Args&... args) {
+  const auto& arg = get<N>(args...);
+  if constexpr (detail::is_named_arg<remove_cvref_t<decltype(arg)>>()) {
```

Comment:
Why remove_cvref instead of remove_cv as before?

Reply:
Because it was `remove_cv<T>` with `T` as a template argument. Where `T` is a type of `field` of `spec_field`, created using:

https://github.com/fmtlib/fmt/blob/1d4199f46bf9f13b263e347e7923c3a9b773029f/include/fmt/compile.h#L721

https://github.com/fmtlib/fmt/blob/1d4199f46bf9f13b263e347e7923c3a9b773029f/include/fmt/compile.h#L492-L499

There was no need to have `remove_cvref_t<T>` before, but now I need it because it's always a reference.